<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Responsable</class>
 <widget class="QMainWindow" name="Responsable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface de Gestion</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonProfil">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Mon Profil</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>311</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonClasses">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>CLASSES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModules">
       <property name="text">
        <string>MODULES</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonEtudiants">
       <property name="text">
        <string>ETUDIANTS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFormateurs">
       <property name="text">
        <string>FORMATEURS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
